Formula/emscripten.rb

```diff
@@ -5,8 +5,8 @@ class Emscripten < Formula
   homepage "https://emscripten.org/"
   # TODO: Remove from versioned dependency conflict allowlist when `python`
   #       symlink is migrated to `python@3.10`.
```

Comment:
We may want to deal with this, but modifying audit list will have merge conflict with #107877.